apps/sharebymail/lib/ShareByMailProvider.php

```diff
@@ -567,7 +567,10 @@ protected function sendPasswordToOwner(IShare $share, string $password): bool {
 		$initiator = $this->userManager->get($share->getSharedBy());
 		$initiatorEMailAddress = ($initiator instanceof IUser) ? $initiator->getEMailAddress() : null;
 		$initiatorDisplayName = ($initiator instanceof IUser) ? $initiator->getDisplayName() : $share->getSharedBy();
-		$shareWith = $share->getSharedWith();
+		$shareWith = $this->getSharedWithEmails($share);
+		if (is_array($shareWith)) {
+			$shareWith = implode(', ', $shareWith);
+		}
```

Comment:
```suggestion
		$shareWith = implode(', ', $this->getSharedWithEmails($share));
```
You don't need to check if it's an array, it always is (like psalm also tells you).